application/common/models/Reset.php

```diff
@@ -146,23 +146,23 @@ public function send()
 
     public function sendPrimary()
     {
-        /**
-         * @todo send email to $user->email with reset code
+        /*
+         * send email to $user->email with reset code
```

Comment:
I created issues in github instead, the @todo impacts the score in scrutinizer and they aren't a very good way to keep track of work needed anyway